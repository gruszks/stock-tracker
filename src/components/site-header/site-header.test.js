import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';

import SiteHeader from './site-header';

it('renders correctly', () => {
  const { asFragment } = render(
    <Router>
      <SiteHeader />
    </Router>
  );

  expect(asFragment()).toMatchSnapshot();
});
