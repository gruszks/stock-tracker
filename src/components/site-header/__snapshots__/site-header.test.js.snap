// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <header
    class="site-headerstyles__Header-ml592c-0 fRAUiU"
  >
    <h1
      class="site-headerstyles__Logo-ml592c-1 jEJIxJ"
    >
      <a
        href="/companies/"
      >
        Stock Tracker
      </a>
    </h1>
    <nav>
      <ul
        class="site-headerstyles__Menu-ml592c-2 gniZBi"
      >
        <li>
          <a
            href="/companies/add/"
          >
            Track new company
          </a>
        </li>
        <li>
          <a
            href="/companies/"
          >
            Companies
          </a>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
